io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -851,7 +858,7 @@ impl<'n> Nexus<'n> {
             let name = self.name.clone();
 
             // After calling unregister_bdev_async(), Nexus is gone.
-            let evt = self.event(EventAction::Delete);
+            let evt = Event::event(&(*self), EventAction::Delete);
```

Comment:
Why this change is needed?

Reply:
I had to use the event() function from two different traits Event and EventWithMeta in the same module. I added this to use event() fn from Event trait.

Reply:
Perhaps, deref might be neater, but just a nit:
```suggestion
            let evt = Event::event(self.deref(), EventAction::Delete);
```